### PR TITLE
CRSF RSSI% and LQ fix

### PR DESCRIPTION
**Pull request by OptimusTi**
This initializes and sets RSSI% and LQ to zero when there's no connection. Fixes issue with setup page of the Configurator and disarm stats too.

### CODE REVIEW

**Comment by avsaase** [comment]
I'm not too familiar with how the protocol works but I think ExpressLRS still reports LQ during RX loss. Would this change set LQ to zero in that situation?

Paging @CapnBry, what do you think?

**Comment by CapnBry** [comment]
We (ExpressLRS) do report linkstats before connection, after disconnection if LOCK_ON_FIRST_RATE is not set, and after failsafe until we disconnect (between 2.5 and 4 seconds depending on rate) but on the last one the values are wrong currently because they don't update if we stop sending packets. I think that last part will be fixed in the Phase Lock PR when that gets merged (or at least it was planned). 

So mostly we do not send updates when disconnected unless we're changing rates. Note that our RSSI can report as low as -128, and have sensitivity as low as -123 (not -120 that the constrain applies) but I think the RSSI% scale is arbitrary and doesn't matter anyway so I don't think you need to worry about changing that.

We should probably fix it on our side so we send you guys the right information, but you failsafe waaaaaaay before we go to disconnected so I can see the user wanting to see the LQ dropping from us until it hits 0, but it will happen so quickly that I don't think anyone would notice if you set it to 0 immediately.

Also, there's an extra set of curly braces here that doesn't break anything but certainly looks confusing.
https://github.com/iNavFlight/inav/pull/7441/files#diff-a680ca1e23901579e9214c055509004e8b5c05e3f7281ed74ac1edfc38caabaeR3530

**Comment by OptimusTi** [comment]
> We (ExpressLRS) do report linkstats before connection, after disconnection if LOCK_ON_FIRST_RATE is not set, and after failsafe until we disconnect (between 2.5 and 4 seconds depending on rate) but on the last one the values are wrong currently because they don't update if we stop sending packets. I think that last part will be fixed in the Phase Lock PR when that gets merged (or at least it was planned).
> 
> So mostly we do not send updates when disconnected unless we're changing rates. Note that our RSSI can report as low as -128, and have sensitivity as low as -123 (not -120 that the constrain applies) but I think the RSSI% scale is arbitrary and doesn't matter anyway so I don't think you need to worry about changing that.
> 
> We should probably fix it on our side so we send you guys the right information, but you failsafe waaaaaaay before we go to disconnected so I can see the user wanting to see the LQ dropping from us until it hits 0, but it will happen so quickly that I don't think anyone would notice if you set it to 0 immediately.
> 
> Also, there's an extra set of curly braces here that doesn't break anything but certainly looks confusing.
> https://github.com/iNavFlight/inav/pull/7441/files#diff-a680ca1e23901579e9214c055509004e8b5c05e3f7281ed74ac1edfc38caabaeR3530

Thanks for checking guys. The reason I sent this is because crossfire is not dropping to zero after disconnecting and RSSI% was showing a wrong value in the Configurator. As for the sensitivity... I wanted to use -130 but someone disagreed. :) Still, I believe that an RSSI alarm set to 20% should work for most users.

File: src/main/io/osd.c [hunk 1]
```diff
@@ -3513,10 +3527,12 @@ static void osdUpdateStats(void)
     if (stats.min_rssi > value)
         stats.min_rssi = value;
 
-    value = osdGetCrsfLQ();
+    value = osdGetCrsfLQ(); {
```
Comment (DzikuVx) [inline src/main/io/osd.c]: Opening `{` here? I think it's a mistake

File: src/main/io/osd.c [hunk 2]
```diff
@@ -3513,10 +3527,12 @@ static void osdUpdateStats(void)
...
     if (stats.min_lq > value)
         stats.min_lq = value;
-
+    if (!failsafeIsReceivingRxData())
```
Comment (DzikuVx) [inline src/main/io/osd.c:3537]: Maybe `{` should be here?

**Review by DzikuVx — CHANGES_REQUESTED** [review]

File: src/main/io/osd.c (hunk 1 above)
Comment (OptimusTi) [reply src/main/io/osd.c]: Thanks. Sending out the fix.

**Review by DzikuVx — APPROVED** [review]

**Pull request merged**